recipes/graylog-forwarder/recipe.rb

```diff
@@ -0,0 +1,55 @@
+require_relative '../tools'
+
+class GraylogForwarder < FPM::Cookery::Recipe
+  include Tools
+
+  description 'Graylog Forwarder'
+
+  name     'graylog-forwarder'
+  version  data.version
+  revision data.revision
+  homepage data.homepage
+  arch     'all'
+
+  source data.source
+  sha256 data.sha256
+
+  maintainer data.maintainer
+  vendor     data.vendor
+  license    'see https://www.graylog.org/enterprise'
+
+  targets :deb do
+    section 'net'
+    config_files '/etc/graylog/forwarder/forwarder.conf'
+    config_files '/etc/graylog/forwarder/jvm.options'
```

Comment:
This needs to be a list. Otherwise the second call will overwrite the first one.
```suggestion
    config_files '/etc/graylog/forwarder/forwarder.conf', '/etc/graylog/forwarder/jvm.options'
```

Reply:
Thanks, I wasn't aware of that.